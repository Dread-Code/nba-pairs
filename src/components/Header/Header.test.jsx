import React from 'react'
import { shallow } from 'enzyme'
import HeaderPair from './Header'
import { PlayerContext } from '../../Context/PlayersContext'

describe('Header', () => {
  test('should match to snapshoot', () => {
    const wrapper = shallow(
      <PlayerContext.Provider>
        <HeaderPair />
      </PlayerContext.Provider>
    )
    expect(wrapper).toMatchSnapshot()
  })
})
