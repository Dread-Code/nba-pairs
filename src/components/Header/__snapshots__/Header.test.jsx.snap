// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match to snapshoot 1`] = `<HeaderPair />`;
